### PR TITLE
Removed Poltergeist; Added ChromeDriver

### PR DESCRIPTION
**Pull request by vladimirbazhanov**
# Summary

- Removed Poltergeist
- Configured using capybara with chromedriver
- Downraged jQuery version to 2.x because of strage issue with Foundation (will update it again with founation-rails update)

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -8,6 +8,6 @@
 Dir[Rails.root.join("spec", "support", "**", "*.rb")].sort.each { |f| require f }
 
 RSpec.configure do |config|
-  config.use_transactional_fixtures = false
+  config.use_transactional_fixtures = true
```
Comment (timurvafin) [inline spec/rails_helper.rb:11]: What is the reason for this change?

File: spec/support/capybara.rb [hunk 2]
```diff
@@ -1,7 +1,17 @@
-require "capybara/poltergeist"
-
 Capybara.configure do |config|
   config.match = :prefer_exact
-  config.javascript_driver = :poltergeist
   config.asset_host = "http://#{ENV.fetch('HOST')}"
 end
+
+Capybara.register_driver :headless_chrome do |app|
```
Comment (timurvafin) [inline spec/support/capybara.rb:6]: Could we use options from https://github.com/teamcapybara/capybara/blob/master/lib/capybara/registrations/drivers.rb#L27-L36 

* `opts.args << '--disable-gpu' if Gem.win_platform?`
* `opts.args << '--disable-site-isolation-trials'`

File: spec/support/capybara.rb [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+  caps = Selenium::WebDriver::Remote::Capabilities.chrome(loggingPrefs: { browser: "ALL" })
+  opts = Selenium::WebDriver::Chrome::Options.new
+
+  chrome_args = %w[--headless --no-sandbox --disable-gpu --window-size=1920,1080]
+  chrome_args.each { |arg| opts.add_argument(arg) }
+
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options: opts, desired_capabilities: caps)
+end
+
+Capybara.default_driver = :headless_chrome
```
Comment (timurvafin) [inline spec/support/capybara.rb:16]: Do we need to enable this driver as default, will it affect tests speed?

File: spec/rails_helper.rb (hunk 1 above)
Comment (vladimirbazhanov) [reply spec/rails_helper.rb:11]: "Sign In" spec fails when use_transactional_fixtures is FALSE, for now I cant' say actually why it happens.
